Source/Fuse.Scripting.JavaScriptCore/Context.uno

```diff
@@ -5,6 +5,11 @@ using Uno;
 
 namespace Fuse.Scripting.JavaScriptCore
 {
+	[extern(Android) Require("Gradle.Dependency.Compile", "org.webkit:android-jsc:r174650")]
+	[extern(Android) Require("LinkLibrary", "jsc")]
+	[extern(Android) Require("IncludeDirectory", "@(PACKAGE_DIR:Path)/3rdparty/JavaScriptCore/Headers")]
+	[extern(Android) Require("LinkDirectory", "${CMAKE_CURRENT_SOURCE_DIR}/app/build/intermediates/exploded-aar/org.webkit/android-jsc/r174650/jni/${ANDROID_ABI}")]
```

Comment:
I would love to get some tips here... from all the resources I found, Gradle, CMake and NDK doesn't cooperate very well, when handling Gradle packages containing native libraries. The best approach I found was to find the library in the exploded-aar folder.